src/main/resources/db/migration/V1_4__add_sync_audit_new_tables copy.sql

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
please change the file name, has space also inn file name

Reply:
this script already added in the demo env and if i change the name then checksum issue will face.